platform/kobo/koreader.sh

```diff
@@ -80,6 +80,9 @@ if [ ! -n "${PLATFORM}" ] ; then
 fi
 # end of value check of PLATFORM
 
+# Remount SD to RW, ignore errors since we may not have sd card
```

Comment:
I surely can implement a more complex logic, i.e.

mount | grep -P ' /mnt/sd .*rw'
if [ $? ]
then
mount -o remount,rw /mnt/sd
fi

But it seems the change won't make things better. We may still fail to mount the partition as RW, which has exactly same possibility as current solution.

What do you think?